src/components/Footer/Footer.tsx

```diff
@@ -0,0 +1,74 @@
+import classNames from 'classnames';
+import React from 'react';
+import { Filter } from '../../types/Filter';
+
+type Props = {
+  onSetFilterType: React.Dispatch<Filter>,
```

Comment:
type is wrong here. would be worth to change:
```suggestion
  onSetFilterType: (Filter) => void,
```